src/components/v2/Layout/Toolbar/styles.ts

```diff
@@ -6,8 +6,9 @@ export const useStyles = () => {
   return css`
     padding-top: ${theme.spacing(3)};
     padding-bottom: ${theme.spacing(3)};
-    [${theme.breakpoints.up('md')}]: {
-      min-height: 96;
+    min-height: 96;
+    [${theme.breakpoints.down('md')}]: {
```

Comment:
in order of mobile first i'd use only `breakpoints.up` (but it's just recommendation)